defaults/config.js

```diff
@@ -30,7 +30,7 @@ module.exports = {
 	// Set the local IP to bind to.
 	//
 	// @type     string
-	// @default  "0.0.0.0"
+	// @default  undefined
```

Comment:
Yeah, that's probably a good plan. Just a short explanation that undefined means it'll listen on everything.